lib/src/opent_id_connect_credentials.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:gotrue/gotrue.dart';
+/// `idToken` is the IDToken of the user.
+///
+/// `nonce` is the nonce of the user.
+///  The nonce is a random value generated by the developer (= yourself)
+///  before the initial grant is started.
+///  You should check the OpenID Connect specification for details.
+///  https://openid.net/developers/specs/
+///
+/// `provider` is the provider of the user.
+///
+/// `clientID` is the clientID of the user.
+///
+/// `issuer` is the issuer of the user.
```

Comment:
The documentation should be at each field

Reply:
Like right each field?